spec/cb/responses/resumes/resume_list_spec.rb

```diff
@@ -11,8 +21,8 @@
   context 'when the Api response is missing the results field' do
     it 'raises an exception' do
       response.delete('Resumes')
-      expect{ Cb::Responses::ResumeList.new(response) }.
-          to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
+      expect { Cb::Responses::ResumeList.new(response) }
+        .to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
```

Comment:
Last one I'm going to comment on, they stand out when scrolling through the PR from here if you want to tackle them.